app/lib/govuk_elements_form_builder/form_builder.rb

```diff
@@ -95,8 +95,8 @@ def govuk_radio_button(attribute, value, *args)
     # You can pass an error with the :error parameter.
     # e.g., <%= form.govuk_collection_radio_buttons(:gender, ['f', 'm'], error: 'Please select a gender') %>
     #
-    # If you wish to specify the size of the heading pass a hash into title with text and size:
-    # <%= form.govuk_collection_radio_buttons(:gender, ['f', 'm'], title: { text: 'What is your gender?', size: :m } ) %>
+    # If you wish to specify the size of the heading and/or which header tag to use, pass a hash into title with text and size:
+    # <%= form.govuk_collection_radio_buttons(:gender, ['f', 'm'], title: { text: 'What is your gender?', size: :m, htag: :h2 } ) %>
```

Comment:
Actually this is my bad - but perhaps add that the default if no htag is supplied is h1